lib/signex.ex

```diff
@@ -146,4 +153,41 @@ defmodule SignEx do
     |> String.downcase()
   end
 
+
+  @spec sign_message(binary, digest :: atom, keypair :: map)
+  :: {:ok, {key_id :: String.t, signature :: String.t}} | {:error, atom}
+
+  def sign_message(message, digest, %{public_key: public_key, private_key: private_key}) do
+    if digest in Algorithm.available_digests() do
+      true = digest in Algorithm.available_digests()
```

Comment:
can kill this line I think

Reply:
![](https://media.giphy.com/media/8WdsK61D9YOOc/giphy.gif)